### PR TITLE
Improve Docs

### PR DESCRIPTION
**Pull request by frankreno**
###### Description

Fill in your description here.
Improve non-helm install docs with more detailed steps and move filter/trim/relabel/custom metrics docs to standalone documentation.

###### Testing performed

- [ ] ci/build.sh
- [ ] Redeploy fluentd and fluentd-events pods
- [ ] Confirm events, logs, and metrics are coming in

### CODE REVIEW

File: deploy/docs/Non_Helm_Installation.md [hunk 1]
```diff
@@ -202,12 +205,26 @@ __NOTE__ It’s fine to change the value of the `cluster` field, but don’t cha
...
 __NOTE__ If you plan to install Prometheus in a different namespace than you deployed Fluentd to in Step 1, or you have an existing Prometheus you plan to apply our configuration to running in a different namespace,  please update the remote write API configuration to use the full service URL like, `http://collection-sumologic.sumologic.svc.cluster.local:9888`.
 
-You can also [Filter metrics](#filter-metrics) and [Trim and relabel metrics](#trim-and-relabel-metrics) in `prometheus-overrides.yaml`.
+You can also [Filter metrics](additional_prometheus_configuration.md#filter-metrics) and [Trim and relabel metrics](additional_prometheus_configuration.md#trim-and-relabel-metrics) in `prometheus-overrides.yaml`.
 
 Install `prometheus-operator` by generating the yaml files using Helm:
 
 ```bash
-$ helm template stable/prometheus-operator --name prometheus-operator --set dryRun=true -f prometheus-overrides.yaml > prometheus.yaml
+<<<<<<< HEAD
```
Comment (samjsong) [inline deploy/docs/Non_Helm_Installation.md]: There's some merge conflicts here, I see we have the other `helm fetch` command on line 189 above that conflicts with line 214 here

File: deploy/docs/Non_Helm_Installation.md [hunk 2]
```diff
@@ -219,6 +236,12 @@ kubectl -n sumologic logs prometheus-prometheus-operator-prometheus-0 prometheus
 
 At this point setup is complete and metrics data is being sent to Sumo Logic.
 
+<<<<<<< HEAD
+__NOTE__ You can also [send custom metrics](additional_prometheus_configuration.md#custom-metrics)to Sumo Logic from Prometheus.
```
Comment (samjsong) [inline deploy/docs/Non_Helm_Installation.md]: Merge conflicts here as well

File: deploy/docs/Non_Helm_Installation.md [hunk 3]
```diff
@@ -230,210 +253,89 @@ kubectl -n kube-system patch service prometheus-operator-kube-controller-manager
...
-### Metrics
 
-#### Filter metrics
+## Deploy FluentBit
 
-The `prometheus-overrides.yaml` file specifies metrics to be collected. If you want to exclude some metrics from collection, or include others, you can edit `prometheus-overrides.yaml`. The file contains a section like the following for each of the Kubernetes components that report metrics in this solution: API server, Controller Manager, and so on.
+In this step, you will deploy FluentBit to forward logs to Fluentd.
 
-If you would like to collect other metrics that are not listed in `prometheus-overrides.yaml`, you can add a new section to the file.
+<<<<<<< HEAD
```
Comment (samjsong) [inline deploy/docs/Non_Helm_Installation.md]: Merge conflicts here as well

File: deploy/docs/Non_Helm_Installation.md (hunk 1 above)
Comment (frankreno) [reply deploy/docs/Non_Helm_Installation.md]: sigh, something is really messed up with my Git. Thanks @samjsong

File: deploy/docs/Non_Helm_Installation.md (hunk 3 above)
Comment (frankreno) [reply deploy/docs/Non_Helm_Installation.md]: fixed

File: deploy/docs/Non_Helm_Installation.md (hunk 2 above)
Comment (frankreno) [reply deploy/docs/Non_Helm_Installation.md]: fixed

File: deploy/docs/Non_Helm_Installation.md (hunk 1 above)
Comment (frankreno) [reply deploy/docs/Non_Helm_Installation.md]: fixed

**Pull request merged**